daemon/api.go

```diff
@@ -698,7 +698,7 @@ func sideloadSnap(c *Command, r *http.Request) Response {
 	contentType := r.Header.Get("Content-Type")
 
 	if strings.HasPrefix(contentType, "multipart/") {
-		// spec says POSTs to sideload snaps should be “a multipart file upload”
+		// spec says POSTs to sideload snaps should be "a multipart file upload"
```

Comment:
I'd prefer to use ASCII consistently. It's not about compatibility (I think even windows will handle that nowadays) but about convenience of being able to copy-paste stuff around and see it work.